phys/module_diag_trad_fields.F

```diff
@@ -137,6 +160,22 @@ SUBROUTINE trad_fields ( u,v,w,t,qv,zp,zb,pp,pb,p,pw,            &
 
                sealevelp(i,j) =   MSLP ( ht(i,j), pressure(i,kms,j), geoheight(i,kms,j) , &
                                          qv(i,kms,j), temperature(i,kms,j) )
+
+               ! Total rainfall
+
+               rain(i,j) =   rainc(i,j) + rainnc(i,j)
+
+               ! Total liquid rainfall
+
+               liqrain(i,j) =   rainc(i,j) + rainnc(i,j) - snownc(i,j) - graupelnc(i,j) - hailnc(i,j)
+
+               ! Total precipitable water
+
+               tpw(i,j) = PWAT(kme-kms+1, qv(i,kms:kme,j), qc(i,kms:kme,j), dz8w(i,kms:kme,j), rho(i,kms:kme,j))  
+
+               ! Surface Pressure
+
+               psfc2d(i,j) =   psfc(i,j) 
```

Comment:
@dudhia @weiwangncar 
Folks,
Is psfc a valid field to use? I do not see where it is updated after input.
Is this a better computation (in start_em.F):
```
2336 ! update surface pressure PSFC (needed for post-processing):
2337                z0 = grid%ph0(i,1,j)/g
2338                z1 = 0.5*(grid%ph0(i,1,j)+grid%ph0(i,2,j))/g
2339                z2 = 0.5*(grid%ph0(i,2,j)+grid%ph0(i,3,j))/g
2340                w1 = (z0 - z2)/(z1 - z2)
2341                w2 = 1. - w1
2342                grid%psfc(i,j) = w1*(grid%p(i,1,j)+grid%pb(i,1,j))+w2*(grid%p(i,2,j)+grid%pb(i,2,j))
```

Reply:
@davegill PSFC is updated every time step in module_surface_driver.F for physics use. Should not need to create a new field, psfc2d, for output, though.